### PR TITLE
COS-2947: Add sysstat extension support

### PR DESCRIPTION
**Pull request by aaradhak**
Ref: https://issues.redhat.com/browse/COS-2947
Related to https://github.com/openshift/os/pull/1621

**- What I did**
This PR adds `sysstat` to the list of extensions.

### CODE REVIEW

File: pkg/daemon/update.go [hunk 1]
```diff
@@ -1775,6 +1775,7 @@ func getSupportedExtensions() map[string][]string {
 		"kerberos":             {"krb5-workstation", "libkadm5"},
 		"kernel-devel":         {"kernel-devel", "kernel-headers"},
 		"sandboxed-containers": {"kata-containers"},
+		"sysstat":		{"sysstat"},
```
Comment (dustymabe) [inline pkg/daemon/update.go:1778]: spacing here probably needs to match

```suggestion
		"sandboxed-containers": {"kata-containers"},
		"sysstat":		        {"sysstat"},
```

Comment (aaradhak) [reply pkg/daemon/update.go:1778]: I was actually going back & forth on this spacing as the outputs were different in vi and vscode editors. Guess I should have trusted vscode here.

**Pull request merged**